src/classes/Options.ts

```diff
@@ -595,23 +599,23 @@ export function loadOptions(options?: Options): Options {
         bptfAccessToken: getOption('bptfAccessToken', '', String, incomingOptions),
         bptfAPIKey: getOption('bptfAPIKey', '', String, incomingOptions),
 
-        admins: getOption('admins', [], JSON.parse, incomingOptions),
-        keep: getOption('keep', [], JSON.parse, incomingOptions),
-        groups: getOption('groups', ['103582791464047777', '103582791462300957'], JSON.parse, incomingOptions),
-        alerts: getOption('alerts', ['trade'], JSON.parse, incomingOptions),
+        admins: getOption('admins', [], jsonParseArray, incomingOptions),
+        keep: getOption('keep', [], jsonParseArray, incomingOptions),
+        groups: getOption('groups', ['103582791464047777', '103582791462300957'], jsonParseArray, incomingOptions),
+        alerts: getOption('alerts', ['trade'], jsonParseArray, incomingOptions),
 
         pricestfAPIToken: getOption('pricestfAPIToken', '', String, incomingOptions),
 
-        skipBPTFTradeofferURL: getOption('skipBPTFTradeofferURL', true, JSON.parse, incomingOptions),
-        skipAccountLimitations: getOption('skipAccountLimitations', true, JSON.parse, incomingOptions),
-        skipUpdateProfileSettings: getOption('skipUpdateProfileSettings', true, JSON.parse, incomingOptions),
+        skipBPTFTradeofferURL: getOption('skipBPTFTradeofferURL', true, jsonParseBoolean, incomingOptions),
+        skipAccountLimitations: getOption('skipAccountLimitations', true, jsonParseBoolean, incomingOptions),
+        skipUpdateProfileSettings: getOption('skipUpdateProfileSettings', true, jsonParseBoolean, incomingOptions),
 
         timezone: getOption('timezone', '', String, incomingOptions),
         customTimeFormat: getOption('customTimeFormat', '', String, incomingOptions),
         timeAdditionalNotes: getOption('timeAdditionalNotes', '', String, incomingOptions),
 
-        debug: getOption('debug', true, JSON.parse, incomingOptions),
-        debugFile: getOption('debugFile', true, JSON.parse, incomingOptions)
+        debug: getOption('debug', true, jsonParseBoolean, incomingOptions),
+        debugFile: getOption('debugFile', true, jsonParseBoolean, incomingOptions)
```

Comment:
What is the point of these parsings? The validator on [628](https://github.com/TF2Autobot/tf2autobot/pull/238/files#diff-b908d13062e3357d9eeab430cb13c1ee6398ebd24d3b6d8d2d93ee6a9ba56720R628) would pick up anything otherwise right?

Reply:
TypeScript any.

Reply:
🤯